src/coffee/helpers/repeater.coffee

```diff
@@ -57,7 +57,9 @@ class Repeater
 
     debug '(re)-trying task, %d remaining attempts', remainingAttempts
     if remainingAttempts is 0
-      defer.reject new Error "Failed to retry the task after #{@_options.attempts} attempts. Cause: #{lastError.stack}"
+      defer.reject
```

Comment:
Oh, I see. Make sense now.

This will be a breaking change, thinking how possible for this to be backward compatible